src/databases/Sqlite.ts

```diff
@@ -12,37 +12,35 @@ export class Sqlite implements IDatabase {
     {
     }
 
-    prepare(type: QueryType, query: string, params: any[]) {
+    async prepare(type: QueryType, query: string, params?: any[]) {
         const preparedQuery = this.db.prepare(query);
 
         switch (type) {
             case 'get': {
-                return preparedQuery.get(...params);
+                if (params) {
+                    return preparedQuery.get(...params);
+                } else {
+                    return preparedQuery.get();
+                }
```

Comment:
These two operations should be equivalent. Why are they being split up?

Reply:
"exploding` can't be done when the object is undefined. It gives an error.

Reply:
Ah. In that case, you can either use a default argument value: `params?: any[] = {}`

Or use an operator: `params = params ?? []` or `params = params || []`